lib/inc/drogon/WebSocketConnection.h

```diff
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include <json/value.h>
```

Comment:
I think you could get away with just use a forward declaration here.

Reply:
I'm concerned with the change of the JSON library in the future, will we remember and detect that the JSON forward declaration is existent in this header file?
I'll wait for the maintainers to give a comment on this before going ahead with the change